controllers/signup.js

```diff
@@ -34,4 +34,44 @@ const doesUSNExist =  (usn) => {
     });
 }
 
-module.exports = doesUSNExist;
\ No newline at end of file
+/**
+ * Function to generate Random Password
+ * @param {Length of the password with maximum 10} length
+ * @returns String
+ */
+const generatePassword = (length) => { 
+    const randomstring = Math.random().toString(36).slice(-length); 
+    return randomstring
+}
+
+/**
+ * Insert student details into database while registration
+ * @param {JSON object containing student details} user
+ * @returns Promise<boolean>
+ */
+const addStudent = (user) => {
+    const client = new pg.Client(config);
+    var addedStudent = false;
+    return new Promise((resolve, reject) => {
+        client.connect()
+        .then(() => {
+            client.query(`INSERT INTO Stu_Per_Data (USN ,Name,Email, Parent_name, Par_Mobile_No, Mobile_No, latitude, longitude,password)
+            VALUES ('${user.usn}','${user.name}', '${user.email}', '${user.parentName}', '${user.parentContact}', '${user.contact}',${user.latitude}, ${user.longitude}, '${generatePassword(8)}');`)
```

Comment:
Ohh Okay, the password will be generated at the time of submitting the signup form
Am I right??

Reply:
Yes. But, user doesn't know the password